core/src/test/scala/kafka/metrics/ClientMetricsRequestReponseTest.scala

```diff
@@ -69,22 +85,21 @@ class ClientMetricsRequestResponseTest {
 
     // Push interval must be set to the default push interval and requested metrics list should be empty
     assertTrue(response.pushIntervalMs() == ClientMetrics.DEFAULT_PUSH_INTERVAL &&
-               response.pushIntervalMs() != sgroup1.getPushIntervalMs)
+               response.pushIntervalMs() != subscription1.getPushIntervalMs)
     assertTrue(response.subscriptionId() == cmClient.getSubscriptionId)
     assertTrue(response.requestedMetrics().isEmpty)
 
     // Now create a client subscription with client id that matches with the client.
     val props = new Properties()
-    val clientMatch = List(s"${CmClientInformation.CLIENT_SOFTWARE_NAME}=Python",
-                           s"${CmClientInformation.CLIENT_SOFTWARE_VERSION}=11.1.*")
+    val clientMatch = List(s"${CLIENT_SOFTWARE_NAME}=Python", s"${CLIENT_SOFTWARE_VERSION}=11.1.*")
     props.put(ClientMetricsConfig.ClientMetrics.ClientMatchPattern, clientMatch.mkString(","))
-    val sgroup2 = createCMSubscriptionGroup("cm_2", props)
-    assertTrue(sgroup2 != null)
+    val subscription2 = createCMSubscription("cm_2", props)
+    assertTrue(subscription2 != null)
```

Comment:
assertNotNull? below as well.

Reply:
@philipnee there are many places where I had to change the assertTrue to assertNull , since it is just a preference of usage of the assert method, for now I am not going to make those changes.. perhaps in future PR's I will remember to use assertNull.